app/controllers/admin/volunteers.py

```diff
@@ -55,7 +55,7 @@ def manageVolunteersPage(eventID):
 
     isProgramManager = g.current_user.isProgramManagerForEvent(event)
     bannedUsers = [row.user for row in getBannedUsers(event.program)]
-    if not (g.current_user.isCeltsAdmin or (g.current_user.isCeltsStudentStaff and isProgramManager)):
+    if not (g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentAdmin or (g.current_user.isCeltsStudentStaff and isProgramManager)):
```

Comment:
Same as above. the `isCeltsStudentStaff` check is redundant with `isProgramManager`